src/Generator/DcaFieldGenerator.php

```diff
@@ -231,6 +231,14 @@ public static function getVideoFields()
                 ],
                 'sql' => 'blob NULL',
             ],
+            'videoAlternativeText' => [
```

Comment:
Sind die Texte so lang, dass es ein Text-Feld braucht?

Reply:
Ja es sind die Transkriptionstexte, also komplett was im Video gesagt/gemacht wird.

Reply:
Sollte man das Feld dann nicht lieber transcriptTexts oder so nennen? Oder ist das die offizielle Bezeichnung? Bei "Alternative Text" denke ich an das alt-Attribute bei Bilder, wo dann drin steht "Logo von Firma X"

Reply:
Ja das ist so wie der alt-Attribut für Bilder, taucht dann in der Seite auf wenn das Video nicht dargestellt werden kann. Transcript ist ja Untertitel, und das haben wir schon eingebaut.